contracts/axelar-gateway/src/contract.rs

```diff
@@ -8,6 +8,8 @@ use crate::storage_types::{DataKey, MessageApprovalKey, MessageApprovalValue};
 use crate::{auth, event};
 
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+pub const INSTANCE_TTL_THRESHOLD: u32 = 120_960; // approx 7 days
+pub const INSTANCE_TTL_EXTEND_TO: u32 = 1_036_800; // approx 60 days
```

Comment:
```suggestion
pub const INSTANCE_TTL_THRESHOLD: u32 = 7 * 24 * 3600; // approx 7 days
pub const INSTANCE_TTL_EXTEND_TO: u32 = 60 * 24 * 3600; // approx 60 days
```

Reply:
We have to account for an approximate ledger time of 5 seconds, so the calculation is actually ((7 * 24 * 60 * 60) / 5)